src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,197 @@
 package org.example;
 
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+import java.util.Scanner;
+
+import static org.example.ConsoleColors.*;
+
 public class Main {
+    private static Scanner scan;
```

Comment:
CLEAN-CODE: lepsi pojmenovani promenne je primo scanner je to vice popisne.
```suggestion
    private static Scanner scanner;
```